TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/SourceTargetCaptureTuple.java

```diff
@@ -27,6 +31,8 @@
 @Slf4j
 public class SourceTargetCaptureTuple implements AutoCloseable {
     public static final String OUTPUT_TUPLE_JSON_LOGGER = "OutputTupleJsonLogger";
+    private static final MetricsLogger metricsLogger = new MetricsLogger("SourceTargetCaptureTuple");
```

Comment:
It would be a lot cleaner if this were decomposed into separate responsibilities.  Here's what we do with the raw requests/responses.
Some of those handlers may convert these to higher order messages.
When we have higher order messages, we should be able to attach consumers/listeners on top of those.
Right now, it's exceptionally easy to see how metrics would be completely suppressed if we decided NOT to log tuple outputs.